<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FPolz</class>
 <widget class="QWidget" name="FPolz">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Список пользователей</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/1/group.png</normaloff>:/res/1/group.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnAdd">
         <property name="text">
          <string>Добавить</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/res/1/group_add.png</normaloff>:/res/1/group_add.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDel">
         <property name="text">
          <string>Удалить</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/res/1/group_delete.png</normaloff>:/res/1/group_delete.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnUpd">
         <property name="text">
          <string>Редактировать</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/res/1/group_edit.png</normaloff>:/res/1/group_edit.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
